app/questionnaire/questionnaire_schema.py

```diff
@@ -130,6 +130,42 @@ def get_questions_for_block(block_json):
         for question_json in block_json.get('questions', []):
             yield question_json
 
+    @staticmethod
+    def is_summary_section(section):
+        """Summary section is a section with a single Summary group
+        """
+        if section['groups']:
+            return QuestionnaireSchema.is_summary_group(section['groups'][0])
```

Comment:
I think we should check all groups in the section, as we don't currently restrict a summary to be in its own section. I don't think.

Reply:
@jonnyshaw89 Given that we now have a section summary. Perhaps we should add this restriction? I think I put in this assumption for that reason.

Reply:
rename it to `is_final_summary_section` and check the block type is `Summary` not `SectionSummary`?